packages/playground/blueprints/src/lib/steps/install-asset.ts

```diff
@@ -33,7 +33,7 @@ export async function installAsset(
 	const assetNameGuess = zipFileName.replace(/\.zip$/, '');
 
 	const wpContent = joinPaths(await playground.documentRoot, 'wp-content');
-	const tmpDir = joinPaths(wpContent, crypto.randomUUID());
+	const tmpDir = joinPaths(wpContent, randomString(46));
```

Comment:
[ Question ] Why we increased the characters we are using in comparison with UUID? 
UUID has 36 + 4 characters instead of 46.
Can we change that to be 40 instead?

Reply:
Thanks for the correction. UUID actually is 36 characters long. Example `'d6c556d6-32b8-4bdf-b338-20a16e9860aa'`. 

> A string containing a randomly generated, 36 character long v4 UUID.
https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

I added a default length of 36 to `randomString` function.

https://github.com/WordPress/wordpress-playground/pull/1016/files#diff-0a926c4fcfdcd984fa42eaa66f44f2a5aea8bcde6576ee5ba04698402489ad65R1